fiftyone/utils/sam.py

```
@@ -585,15 +584,11 @@ class SAMSegmenterOutputProcessor(fout.OutputProcessor):
     """Converts SAM model outputs to FiftyOne format.
 
     Args:
-        classes (None): the list of class labels for the model
+        classes (None): the list of class labels for the model. Not used in SAM output processor.
         mask_thresh (0.5): Threshold for converting float masks to boolean masks
     """
 
     def __init__(self, classes=None, mask_thresh=0.5, **kwargs):
```

Ruff (0.15.11)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)